client/adjudicate.go

```diff
@@ -396,3 +409,55 @@ func (c *Channel) subChannelStateMap() (states channel.StateMap, err error) {
 	})
 	return
 }
+
+// ensureRegistered ensures that the channel is registered.
+func (c *Channel) ensureRegistered(ctx context.Context) (err error) {
+	phase := c.Phase()
+	if phase == channel.Registered || phase == channel.Progressed {
+		return
```

Comment:
```suggestion
		return nil
```

This is a minor suggestion.

Generally, I feel it is easier to follow when you use explicit returns instead of named returns.

In the latter case, I will have to go up to the function definition to see the return variables and then figure where these variables have been set. 
But when using explicit returns, most information is contained within that one line.